Sources/Tracking/CustomerIOImplementation.swift

```diff
@@ -206,6 +217,8 @@ class CustomerIOImplementation: CustomerIOInstance {
         name: String,
         data: RequestBody
     ) {
+        CIODataPipeline.analytics.screen(title: name, properties: jsonAdapter.toJson(data))
```

Comment:
Same comment as above with expecting that this CDP call goes inside of the 1 common `trackEvent` function.

Reply:
same comment as above.